### DISCUSSION TITLE
Support for web?

### DISCUSSION THREAD

**Discussion by thecannabisapp** [GetStream/stream-chat-react-native#1411] [Q&A] [answered]
I've seen some old closed issues from 2021 and I'm not sure if this has been resolved. Just wanted clarification on the best route to implementing stream chat for my React Native app which I'm building using Expo and currently is only being released on Web for MVP testing purposes.

I noticed there aren't any C, Swift, or Java code in the repo so I assume there are no native dependencies. Which package is best suited for compatibility with Web, iOS and Android? I'd rather install one package than have to split my code to support web, and then mobile.

Any advice and previous experience in the matter are greatly appreciated! 🙏

**Comment by khushal87** [comment, marked as answer]
You can use stream-chat-react-native to have support for android and iOS platforms. If I understand your question correctly, that's about which SDK should you use to have android, ios as well as web support right?

We do not support web in React Native SDK as of now, but we are already working on a POC around that and plan to take this up in the future and support it. Thanks :)
